improved local node.js installation instructions

keydon opened the pull request:
+ also added systax highlighting for listings
+ simplified image references

This closes OmerTu/GoogleHomeKodi#108

keydon commented:
mhh lets better delay my pull-request until we get those other two almost rdy pull requests through :)

keydon commented:
now is up to date with the recent changes

p0psicles commented on README.md:
```diff
@@ -170,20 +197,79 @@ SyslogIdentifier=nodejs-example
+There are two ways of implpementing the scenario of multiple kodi instance (i.e one in the living room and another in the bedroom).
+1. Having two IFTTT-Applets per command with different phrases. Each applet is responsible for one instance. (i.e. the phrase `living room kodi play...` could target `https://mylivingroom.glitch.com/playpause` and the phrase `bedroom kodi play...` could target `https://bedroom.glitch.com/playpause`).
+2. When running a local node server you can utilize the kodi-hosts.config.js configuration.
+This configuration will allow you to address multiple kodi installations (e.g. "Hey Google, kodi play [movie name] in the bedroom") from a single IFTTT-Applet.
```
Your right it's still two applets. Only now they have the have the same URL endpoint, but different payload.

keydon commented:
fixed you comments.

and for the multiple instances, i wrote something completly diffrent than i wanted to ;O
Now also rewritten. I left out the part that simple commands could be implemented with a single applet, it just confuses ppl.

OmerTu commented:
Looks great, thanks!
